### PR TITLE
updated backupManualPlace_State.cpp

### LINKED ISSUE #223: Note Manager backupManualPlace State Transition
In backupManualPlace_State.cpp
Update IsTransitionCondition() and AtTarget() methods to accomplish the state transitions below.

Return
MANUAL_MODE Button Pressed AND Place Mode
OR 
 (m_mechanism->GetCurrentState() == backupManaulLaunch && m_mechanism->isPlacerMode())

Transition into state

To access robot state and check if it is in placer/launcher mode use the code below
im_mechanism->isPlacerMode()

### PR DESCRIPTION
**Pull request by ImAPenguin56**
closes #223

### CODE REVIEW

**Review by Tan560 — APPROVED** [review]

File: src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualPlace_State.cpp [hunk 1]
```diff
@@ -65,9 +64,8 @@ bool noteManagerbackupManualPlaceState::AtTarget()
 	return attarget;
 }
 
-bool noteManagerbackupManualPlaceState::IsTransitionCondition ( bool considerGamepadTransitions )
+bool noteManagerbackupManualPlaceState::IsTransitionCondition(bool considerGamepadTransitions)
 {
 	// To get the current state use m_mechanism->GetCurrentState()
-
-	return ( considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed ( TeleopControlFunctions::EXAMPLE_MECH_FORWARD ) );
+	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::MANUAL_MODE) && m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer);
```
Comment (Tan560) [inline src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualPlace_State.cpp:70]: Add an OR condition of  (m_mechanism->GetCurrentState() == backupManaulLaunch &&  m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer)

**Review by Tan560 — COMMENTED** [review]
Add an OR condition of (m_mechanism->GetCurrentState() == backupManaulLaunch && m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer)

**Review by Tan560 — CHANGES_REQUESTED** [review]
Add an OR condition of (m_mechanism->GetCurrentState() == backupManaulLaunch && m_mechanism->isPlacerMode())

File: src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualPlace_State.cpp (hunk 1 above)
Comment (Tan560) [reply src/main/cpp/mechanisms/noteManager/decoratormods/noteManager_backupManualPlace_State.cpp:70]: Also update from Main and then replace
m_mechanism->m_scoringMode == RobotStateChanges::ScoringMode::Placer
with
m_mechanism->isPlacerMode()

**Review by Tan560 — CHANGES_REQUESTED** [review]

**Comment by ElliotBayer** [comment]
Deleting branch and creating new one from Main

**Pull request closed without merging**